packages/p2p/src/components/advertiser-page/advertiser-page.scss

```diff
@@ -91,38 +94,120 @@
                 border-right: 1px solid #ededed;
             }
         }
+
+        @include mobile {
+            width: 110vw;
+        }
+
+        &--wrapper {
+            display: flex;
+            align-items: center;
+        }
+
+        .dc-themed-scrollbars__autohide--is-hovered::-webkit-scrollbar-thumb {
+            display: none;
+        }
```

Comment:
Just out of curiosity, what does this do? Would it be more correct to add a new prop to `<ThemedScrollBars>` to future-proof this?

Reply:
Done, woohoo 🙌